article/models.py

```diff
@@ -991,7 +980,7 @@ def get_authors_with_urls(self) -> str:
     authors_with_urls = property(fget=get_authors_with_urls)
 
     def get_authors_in_order(self):
-        AUTHOR_TYPES = ["author", "photographer", "illustrator", "videographer", "coordinating editor", "news editor", "news producer", "features editor", "humour editor", "culture editor", "opinion editor", "video editor", "photo editor", "visuals editor", "sports editor", "science editor"]
+        AUTHOR_TYPES = ["author", "photographer", "illustrator", "videographer"]
         authors = self.article_authors.all()
```

Comment:
These must be added as Anabella has requested them

Reply:
I think its more elegant if we have one of the options to be showing the author's organization role. I've added that. If we think we need to have every organization role in the selector then we can add that